docs/api/IoT.js-API-Assert.md

```diff
@@ -5,203 +5,234 @@ The following shows Assert module APIs available for each platform.
+* `message` {any} Message displayed in the thrown error.
 
-Test if `value` is true, it throws an `AssertionError` exception if `value` evaluates `false`.
+Checks if the `value` is truthy. If it is not, throws an AssertionError, with the given optional `message`.
 
 **Example**
 
 ```js
 var assert = require('assert');
-assert.assert(1 == 1);
+
+assert.assert(1);
+// OK
+
+assert.assert(true);
+// OK
+
+assert.assert(false);
+// throws "AssertionError: false == true"
+
+assert.assert(0);
+// throws "AssertionError: 0 == true"
+
+assert.assert(false, "it's false");
+// throws "AssertionError: it's false"
 ```
 
 
-### `assert.doesNotThrows(block[, message])`
-* `value <Function()>`
-* `message <any>`
+### doesNotThrow(block[, message])
+* `block` {Function}
+* `message` {any} Message to be displayed.
 
-Test if the given `block` does not throw any exception, if not, it throws an exception.
+Tests if the given `block` does not throw any exception, if it does, throws an
+exception with the given optional `message`.
 
 **Example**
 
 ```js
 var assert = require('assert');
+
 assert.doesNotThrow(
   function() {
-    assert.assert(1 == 1);
+    assert.assert(1);
   }
 );
+// OK
+
+assert.doesNotThrow(
+  function() {
+    assert.assert(0);
+  }
+)
+// throws "AssertionError: Got unwanted exception."
 ```
 
 
-### `assert.equal(actual, expected[, message])`
-* `actual <any>`
-* `expected <any>`
-* `message <any>`
+### equal(actual, expected[, message])
+* `actual` {any} The actual value.
+* `expected` {any} The expected value.
+* `message` {any} Message to be displayed.
 
-Test if `actual == expected` is evaluated to `true`, if not, it throws an exception.
+Tests if `actual == expected` is evaluated to `true`, if it is not, throws an
+exception with the given optional `message`.
 
 **Example**
 
 ```js
 var assert = require('assert');
+
 assert.equal(1, 1);
 assert.equal(1, '1');
 ```
 
 
-### `assert.fail(actual, expected, message, operator)`
-* `actual <any>`
-* `expected <any>`
-* `message <any>`
-* `operator <String>`
+### fail(actual, expected, message, operator)
+* `actual` {any} The actual value.
+* `expected` {any} The expected value.
+* `message` {any} Message to be displayed.
+* `operator` {String} The operator.
 
-Throw an `AssertionError` exception.
+Throws an `AssertionError` exception with the given `message`.
 
 **Example**
 
 ```js
 var assert = require('assert');
-assert.fail(1, 2, undefined, '>'); // AssertionError: 1 > 2
+
+assert.fail(1, 2, undefined, '>');
+// AssertionError: 1 > 2
 ```
 
 
-### `assert.notEqual(actual, expected[, message])`
-* `actual <any>`
-* `expected <any>`
-* `message <any>`
+### notEqual(actual, expected[, message])
+* `actual` {any} The actual value.
+* `expected` {any} The expected value.
+* `message` {any} Message to be displayed.
 
-Test if `actual != expected` is evaluated to `true`, if not, it throws an exception.
+Tests if `actual != expected` is evaluated to `true`, if it is not, throws an
```

Comment:
I like shorter sentences, but this is just my preference. I leave this to the other reviewers.

Tests if `actual != expected` is evaluated to `true`. If it is not throws an...